src/prop/prop_engine.h

```diff
@@ -188,7 +188,7 @@ class PropEngine
    * that is definitionally equal to it.  The result of this function
    * is that the Node can be queried via getSatValue().
    */
-  void ensureLiteral(TNode n);
+  Node ensureLiteral(TNode n);
```

Comment:
Document this change

Reply:
The documentation was incorrect before, now it is correct.

Reply:
Hum, my interpretation of the current documentation is that `n` is added to some map (which is done in the cnf stream), not that you produce that as a result. I think that the latter should be said explicitly.

Reply:
I'm not sure I follow.  The documentation states the purpose of the method: it returns a Node that can be queried via getSatValue.